contributors/devel/sig-release/cherry-picks.md

```diff
@@ -62,21 +62,6 @@ your case by supplementing your PR with e.g.,
 - Key stakeholder SIG reviewers/approvers attesting to their confidence in the
   change being a required backport
 
-To illustrate the point, dependency updates that just aim to silence some scanners
-and do not fix any vulnerable code are NOT eligible for cherry-picks.
-
-If the change is in cloud provider-specific platform code (which is in the
```

Comment:
I removed these paragraphs since since cloud-provider-specific code is now out of tree ... I don't *think* we need them any more (but can add them back if reviewers prefer)

Reply:
+1 to leave it out/